src/screens/Profile/EditTariffPlanInformation.tsx

```diff
@@ -70,46 +73,46 @@ function EditTariffPlanInformation(props: EditTarifPlanInformationProps) {
         <BaseScreen title={tariffPlan.name} leftIcon="backNav">
             <List>
                 <ListItemWrapper
-                    leftText='Fixed Payment'
+                    leftText={intl(words.fixed_payment)}
                     rightText={newFixedTariff}
                     editable
                     dense
                     editValue={newFixedTariff}
                     onEditChange={handleNewFixedTariffInput}
                     editInputId={'edit-fixed-tariff'}
-                    editUnit={'Ks'}
-                    editType='number'
+                    editUnit={intl(words.ks)}
+                    editType={intl(words.number)}
```

Comment:
I don't think `editType` needs to be translated. It just restricts the type of input for this